[MRG + 1] docs: Change example of KDTree to prevent DeprecationWarning

jannab opened the pull request:
<!--
Thanks for contributing a pull request! Please ensure you have taken a look at
the contribution guidelines: https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#Contributing-Pull-Requests
-->
#### Reference Issue

Fixes #6906 

<!-- Example: Fixes #1234 -->
#### What does this implement/fix? Explain your changes.
- fill in missing import statement of sklearn.neighbors.KDTree
- wrap X in a list
#### Any other comments?

<!--
Please be aware that we are a loose team of volunteers so patience is
necessary; assistance handling other issues is very welcome. We value
all user contributions, no matter how minor they are. If we are slow to
review, either the pull request needs some benchmarking, tinkering,
convincing, etc. or more likely the reviewers are simply busy. In either
case, we ask for your understanding during the review process.
For more information, see our FAQ on this topic:
http://scikit-learn.org/dev/faq.html#why-is-my-pull-request-not-getting-any-attention.

Thanks for contributing!
-->
- fill in missing import statement of sklearn.neighbors.KDTree
- wrap X in a list

Changes to be committed:
    modified:   sklearn/neighbors/binary_tree.pxi

Resolves: #6906

jannab committed:
docs: Change example of KDTree to prevent DeprecationWarning

- fill in missing import statement of sklearn.neighbors.KDTree
- wrap X in a list

Changes to be committed:
	modified:   sklearn/neighbors/binary_tree.pxi

Resolves: #6906

maniteja123 commented on sklearn/neighbors/binary_tree.pxi:
```diff
@@ -293,11 +293,11 @@ Examples
 
     >>> import numpy as np
-
+    >>> from sklearn.neighbors import KDTree
```
Hi, I am not an expert but looking at the source code and documentation for other ``neighbours``, for eg. [Ball tree](http://scikit-learn.org/stable/modules/generated/sklearn.neighbors.BallTree.html), this would be incorrectly importing ``KDTree`` in the docs. I suppose we can also use the ``{BinaryTree}`` directive used here. Also, it would probably be better to add the import statement for the below examples too. Thanks!

jannab replied:
Hi maniteja123,
thanks for the comment. I'm not sure about the import statement and removed it for now, because it has nothing to do with the issue I attempted to fix about the deprecation warning. If there's something to do about one or more import statement, another Issue should be opened.

jannab committed:
docs: remove import statement of KDTree in example of KDTree

- remove import statement of sklearn.neighbors.KDTree

Changes to be committed:
	modified:   sklearn/neighbors/binary_tree.pxi

Resolves: #6906

amueller commented:
Thanks, LGTM!

agramfort commented:
thx
